integrations/saltstack/saltstack-remediation/sensu-integration.yaml

```diff
@@ -0,0 +1,161 @@
+---
+api_version: catalog/v1
+type: Integration
+metadata:
+  namespace: saltstack
+  name: saltstack-remediation
+spec:
+  class: enterprise
+  provider: remediation
+  display_name: SaltStack
+  short_description: Triggers SaltStack function(s) for automated remediation.
+  tags:
+    - saltstack
+    - remediation
+    - self healing
+    - automation
+  contributors:
+    - "@sensu"
+    - "@nixwiz"
+  prompts:
+    - type: section
+      title: SaltStack API Configuration
+    - type: markdown
+      body: |
+        This integration requires a SaltStack API Host as well as an API Username and Password for API Authentication.
+    - type: question
+      name: saltstack_api_url
+      required: true
+      input:
+        type: string
+        title: SaltStack API URL
+        description: >-
+          Provide the URL Host (i.e. http(s)://hostname or IP address)
+        default: "http://127.0.0.1"
+    - type: question
+      name: saltstack_api_username
```

Comment:
The sensitive questions  need to be updated to include a noop option  along with the secret and envvar options.
The noop options is there if you want to use the env_var preprovisioned in the backend's running environment.
Take a look at 
https://github.com/sensu/catalog/blob/main/integrations/aws/aws-lambda-monitoring/sensu-integration.yaml
for an example.   I'm going to try to make an inline suggestion here as well that follows this pattern

Reply:
Resolved by committing suggested changes.